modules/mapper-extras/src/test/resources/rest-api-spec/test/sparse-vector/10_basic.yml

```diff
@@ -98,3 +98,64 @@ setup:
 - match: {hits.hits.2._id: "1"}
 - gte: {hits.hits.2._score: 0.78}
 - lte: {hits.hits.2._score: 0.791}
+
+---
+"L1 norm":
+- do:
+    headers:
+      Content-Type: application/json
+    search:
+      rest_total_hits_as_int: true
+      body:
+        query:
+          script_score:
+            query: {match_all: {} }
+            script:
+              source: "l1normSparse(params.query_vector, doc['my_sparse_vector'])"
+              params:
+                query_vector: {"2": 0.5, "10" : 111.3, "50": -13.0, "113": 14.8, "4545": -156.0}
```

Comment:
Same here, please add a few more tests that don't only check the behaviour when all dimensions are matching.